mysql scripts/appointmentDB.sql

```diff
@@ -16,6 +16,10 @@ CREATE TABLE Location (
 CREATE TABLE Question (
 	questionId INTEGER PRIMARY KEY NOT NULL AUTO_INCREMENT,
     string VARCHAR(255) NOT NULL,
+    inputType VARCHAR(255) NOT NULL,
```

Comment:
Should we consider having tables with inputTypes, subheadings, and tags? May seem a bit a over-engineered.